apps/coscrad-frontend/src/components/resources/audio-item/audio-item-detail.full-view.presenter.tsx

```diff
@@ -24,23 +79,97 @@ export const AudioItemDetailFullViewPresenter = ({
 
     return (
         <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.audioItem}>
-            <AudioPlayer audioUrl={audioURL} />
-            <SinglePropertyPresenter
-                display="Duration"
-                value={`${convertMillisecondsToSeconds(lengthMilliseconds)} secs`}
-            />
-            <SinglePropertyPresenter display="Audio Url" value={audioURL} />
-            <Grid container columns={6} sx={{ mb: 1 }}>
-                <Grid item xs={1}>
-                    {/* TODO: introduce bold (not heading) custom theme typography variant? */}
-                    <Typography component={'span'} sx={{ fontWeight: 'bold' }}>
-                        Text:
+            {actions.some(({ type: commandType }) => commandType === CREATE_NOTE_ABOUT_RESOURCE) ? (
+                <>
+                    <AudioAnnotator
```

Comment:
Break this block out into a separate component so that it is clear what's going on with the ternary.